test/test_pipeline/test_tabular_regression.py

```diff
@@ -0,0 +1,268 @@
+import os
+import re
+
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    UniformFloatHyperparameter,
+    UniformIntegerHyperparameter,
+)
+
+import numpy as np
+
+import pytest
+
+import torch
+
+from autoPyTorch.pipeline.components.setup.early_preprocessor.utils import get_preprocess_transforms
+from autoPyTorch.pipeline.tabular_regression import TabularRegressionPipeline
+from autoPyTorch.utils.common import FitRequirement
+from autoPyTorch.utils.hyperparameter_search_space_update import HyperparameterSearchSpaceUpdates, \
+    parse_hyperparameter_search_space_updates
+
+
+@pytest.mark.parametrize("fit_dictionary_tabular", ["regression_numerical_only",
+                                                    # "regression_categorical_only",
```

Comment:
This is not supported yet?

Reply:
There is an issue with the CategoricalEncoder implementation, thats why I commented it out for now.

Reply:
Can you tell me what the issue is?

Reply:
I have to check first if it is still there because I think Ravins PR changed something in this regard.

Reply:
Issue seems to be resolved!